core/core-debug.el

```diff
@@ -236,6 +236,20 @@ seconds to load")
                      "You can paste it in the gitter chat.\n"
                      "Check the *Messages* buffer if you need to review it"))))
 
+(defun spacemacs/describe-ex-command (ex-command)
+  (interactive (list (completing-read "Describe ex-command: " evil-ex-commands)))
+  (let* ((func (alist-get ex-command evil-ex-commands nil nil 'string=))
+         (prompt (if (stringp func)
+                     "The ex-command :%s is an alias for the ex-command :%s. Describe :%s?"
+                   "The ex-command :%s calls %s. Describe %s?")))
+    (when (y-or-n-p (format prompt
+                            ex-command
+                            func
+                            func))
+      (if (stringp func)
+          (spacemacs/describe-ex-command func)
+        (describe-function func)))))
```

Comment:
If `helpful` layer is enabled, we can use `(helpful-callable func)` instead of `(describe-function func)`.
The output is much more readable.

Reply:
Thanks for pointing out, I have added it. On the other hand, the helpful layer could make `describe-function` an `alias` for `helpful-callable` (and document it is aliased by the helpful layer in the docstring).

Reply:
Making it an alias, also keeps the `ivy-actions` available. Right now, with the helpful layer, I loose the ivy actions `Info` and `definition` (which I use all the time) when using `SPC h d f`.

Reply:
On the other hand, I already had a personal helpful layer, where I bound the helpful function under `SPC h h f` (etc.) , which is an alternative solution. The command is not called `describe-function` anymore, but `helpful-callable`, so that made more sense to me...

Reply:
I would vote to change those bindings in the helpful layer

Reply:
or provide a configuration variable like `helpful-prefer-alias`

Reply:
I have
```elisp
  (global-set-key (kbd "C-h f") #'helpful-callable)
  (global-set-key (kbd "C-h v") #'helpful-variable)
  (global-set-key (kbd "C-h k") #'helpful-key)
  (global-set-key (kbd "C-c C-d") #'helpful-at-point)
  (global-set-key (kbd "C-h F") #'helpful-function)
  (global-set-key (kbd "C-h o") #'helpful-symbol)
```
in my `user-config`.
The issue with alias is, the first time when you call `describe-*`, it won't automatically use `helpful` package. Because it's not loaded...